src/applications/vaos/sass/vaos.scss

```diff
@@ -19,3 +19,11 @@
 .vaos-form__checkbox > .schemaform-label {
   margin-top: 0;
 }
+
+.vaos-appt-type {
```

Comment:
I was trying to follow a BEM pattern for class names, where `vaos-something` is the name that prepends all our classes as the "block." So this would be more like `vaos-form__appt-type`. Open to other conventions, but it'd be nice to have one we follow consistently.